uobtheatre/bookings/models.py

```diff
@@ -274,10 +274,11 @@ def subtotal(self) -> int:
         Returns:
             int: price of the booking with discounts applied in penies
         """
-        return math.ceil(
-            self.get_best_discount_combination_with_price()[1]
-            * (1 - self.admin_discount_percentage)
-        )
+        if self.performance.has_group_discounts:
```

Comment:
Actually, I feel like this logic should be on the `tickets_discounted_price` function

Reply:
There is no such thing. Not until we support other payable objects (other than just bookings).